app/views/shared/views/_show_containers_dashboard.html.haml

```diff
@@ -0,0 +1,125 @@
+.container-fluid.container-tiles-pf.containers-dashboard.miq-dashboard-view{"ng-app"        => "containerDashboard",
```

Comment:
`.containers-dashboard` in `show.html.haml` but `.single-provider-containers-dashboard` in `_show_dashboard.html.haml`